dragscroll.js

```diff
@@ -1,11 +1,16 @@
 /**
- * @fileoverview dragscroll - scroll area by dragging
- * @version 0.0.8
- * 
- * @license MIT, see http://github.com/asvd/dragscroll
- * @copyright 2015 asvd <heliosframework@gmail.com> 
+ * @fileoverview dragscroll-zoom - scroll area by dragging and zoom in~out
+ * @version 1.0.0
+ *
+ * @license MIT, see https://github.com/cymakr/dragscroll-zoom
+ * @copyright 2017 cymakr <cymakr@gmail.com>
+ *
+ * This project began as a fork of dragscroll https://github.com/asvd/dragscroll
  */
-
+var zoom = 100;
+var sheet = document.createElement('style');
+sheet.innerHTML = "#dragsimg {width:" + zoom + "%;}";
```

Comment:
This should be a class name, not an id (otherwise you can only create a single draggable image)